benchmarking/nursery/benchmark_automl/benchmark_main.py

```diff
@@ -95,10 +91,12 @@ def subsample(
     return transfer_learning_evaluations
 
 
-if __name__ == "__main__":
+def parse_args(methods: dict, benchmark_definitions: dict):
     parser = ArgumentParser()
     parser.add_argument(
-        "--experiment_tag", type=str, required=False, default=generate_slug(2)
```

Comment:
I missed this one in my first review, I will be keen to keep the same behavior (allowing to not pass an experiment name) as it is not preventing from passing an explicit name. 

I can revert this specific one in another PR.